nvflare/lighter/poc_commands.py

```diff
@@ -78,57 +97,359 @@ def get_stop_cmd(poc_workspace: str, service_dir_name: str):
+    if not project_config:
+        raise CLIException(f"empty or invalid project config from project yaml file: {src_project_file}")
+
+    if not user_provided_project_config:
+        project_config = update_server_name(project_config)
+        project_config = update_clients(clients, number_of_clients, project_config)
+        project_config = add_he_builder(use_he, project_config)
+        if docker_image:
+            project_config = update_static_file_builder(docker_image, project_config)
+    save_project_config(project_config, dst_project_file)
+    packages = get_packages(project_config)
+    project = prepare_project(project_config)
+    builders = prepare_builders(project_config)
+    provisioner = Provisioner(workspace, builders)
+    provisioner.provision(project)
+
+    return project_config, packages
+
+
+def get_packages(project_config):
+    packages = {
+        SC.FLARE_SERVER: get_fl_server_name(project_config),
+        SC.FLARE_PROJ_ADMIN: get_proj_admin(project_config),
+        SC.FLARE_CLIENTS: get_fl_client_names(project_config),
+        SC.IS_DOCKER_RUN: is_docker_run(project_config),
+    }
+    return packages
+
+
+def save_project_config(project_config, project_file):
+    with open(project_file, "w") as file:
+        yaml.dump(project_config, file)
+
+
+def update_server_name(project_config):
+    old_server_name = get_fl_server_name(project_config)
+    server_name = "server"
+    if old_server_name != server_name:
+        update_project_server_name_config(project_config, old_server_name, server_name)
+    return project_config
+
+
+def is_docker_run(project_config: OrderedDict):
+    if "builders" not in project_config:
+        return False
+    static_builder = [
+        b
+        for b in project_config.get("builders")
+        if b.get("path") == "nvflare.lighter.impl.static_file.StaticFileBuilder"
+    ][0]
+    return "docker_image" in static_builder["args"]
+
+
+def update_static_file_builder(docker_image: str, project_config: OrderedDict):
+    # need to keep the order of the builders
+    for b in project_config.get("builders"):
+        if b.get("path") == "nvflare.lighter.impl.static_file.StaticFileBuilder":
+            b["args"]["docker_image"] = docker_image
+
+    return project_config
+
+
+def add_docker_builder(use_docker: bool, project_config: OrderedDict):
+    if use_docker:
+        docker_builder = {
+            "path": "nvflare.lighter.impl.docker.DockerBuilder",
+            "args": {"base_image": "python:3.8", "requirements_file": "requirements.txt"},
+        }
+        project_config["builders"].append(docker_builder)
+
+    return project_config
+
+
+def add_he_builder(use_he: bool, project_config: OrderedDict):
+    if use_he:
+        he_builder = {
+            "path": "nvflare.lighter.impl.he.HEBuilder",
+            "args": {},
+        }
+        project_config["builders"].append(he_builder)
+
+    return project_config
```

Comment:
Why do we want HE as default here?
When we get other builders, do we also want to add them here as an option?